user.py

```diff
@@ -1,35 +1,31 @@
-from board import Board, spot_exists, is_occupied, update_state
+from board import Board, SpotStates 
 from ui import Ui
 
 class User:
     current_player = 'X' 
 
-    # players will be able to pick a symbol
     def __init__(self, symbol):
         self.symbol = symbol
 
-def get_input(choice):
-    return input(choice)
+    def switch_current_user(who_moved, opponent):
+        if who_moved == User.current_player:
+            User.current_player = opponent
+            return User.current_player 
 
-def spot_validation(current_state, response):
-    spot_list = []
-    spot_list.extend(range(1, len(current_state) + 1))
-    spot_list = list(map(str, spot_list))
+class UserActions:   
+    def make_move(current_state, response):
```

Comment:
This is a really funky method. it returns both integer and boolean values. I wonder if it can be cleaned up a bit.